core/thread.c

```diff
@@ -25,7 +25,8 @@
 #include "kernel.h"
 #include "irq.h"
 
-#define ENABLE_DEBUG    (0)
+#define ENABLE_DEBUG (0)
```

Comment:
sorry these 3 files were in the PR - if you compare ENABLE_DEBUG within the src tree a majority of lines has " " one space instead of tab or " " for the len of tab.